linera-views/src/test_utils/mod.rs

```diff
@@ -347,6 +348,21 @@ fn get_random_key_values2(num_entries: usize, len_value: usize) -> Vec<(Vec<u8>,
     key_values
 }
 
+/// Add a prefix to a list of key-values
```

Comment:
Nit: verb conjugation according to the convention we use

```suggestion
/// Adds a prefix to a list of key-values
```

Reply:
Ok,